### PR TITLE
RN-1042 Fix media perm Meditrak App

### PR DESCRIPTION
**Pull request by IgorNadj**
### Issue #: RN-1042

Fix in fork: https://github.com/beyondessential/react-native-image-picker/commit/f789975563e09443d03ec6c50fb67bf0fd2522bd

### CODE REVIEW

File: packages/meditrak-app/app/main.js [hunk 1]
```diff
@@ -72,3 +72,6 @@ const App = () => (
 );
 
 AppRegistry.registerComponent('TupaiaMediTrak', () => App);
+
+LogBox.ignoreLogs(['Warning: ...']); // Ignore log notification by message
+LogBox.ignoreAllLogs(); // Ignore all log notifications
```
Comment (rohan-bes) [inline packages/meditrak-app/app/main.js:77]: Is the annoying yellow box a new thing?

**Review by rohan-bes — APPROVED** [review]
Looks good :+1: Thanks for solving this one. It seems it will, but just double checking this is backwards compatible?

File: packages/meditrak-app/app/main.js (hunk 1 above)
Comment (IgorNadj) [reply packages/meditrak-app/app/main.js:77]: This thing at the bottom, just gives PropType warnings, and perfectly covers our survey screen navigation 😆 
image

**Comment by IgorNadj** [comment]
> just double checking this is backwards compatible?

Yep, I have tested and we are doing normal testing on this as well

**Comment by IgorNadj** [comment]
This has gone into master in a separate PR and then into dev with a master -> dev PR

**Pull request closed without merging**